bin/reth/src/args/rpc_server_args.rs

```diff
@@ -99,12 +105,60 @@ impl RpcServerArgs {
             client,
```

Comment:
@gakonst @mattsse for `server` function what is your thought? If just call `launch` function or actually need reimplement similar logic in `RpcServerArgs`?